src/main/java/it/gov/pagopa/pu/organization/model/DistinctCollectionReasonDTO.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.pu.organization.model;
```

Comment:
put all taxonomy Distinct DTO inside taxonomy package
```suggestion
package it.gov.pagopa.pu.organization.model.taxonomy;
```